packages/react-components/src/components/MIMap/GoogleMapsMap/GoogleMapsMap.jsx

```diff
@@ -5,6 +5,7 @@ import 'mapbox-gl/dist/mapbox-gl.css';
 import './GoogleMapsMap.scss';
 import { useIsDesktop } from '../../../hooks/useIsDesktop';
 import isNullOrUndefined from '../../../../../map-template/src/helpers/isNullOrUndefined';
+import { useRecoilState } from 'recoil';
```

Comment:
Revert all changes in the react-components package 🙏

Reply:
👍